### PR TITLE
many: add vendoring of dependencies by default

### PR DESCRIPTION
**Pull request by mvo5**
This is a preview for the vendoring support.

### CODE REVIEW

**Comment by zyga** [comment]
We should also adjust READMEs so that new contributors don't trip over

**Comment by mvo5** [comment]
Indeed, sorry for the missing README.md update. In fact I had it updated, but I forgot to git add, here it is now :)

File: debian/not-installed [hunk 1]
```diff
@@ -0,0 +1 @@
+debian/tmp/usr/bin/uboot-go
```
Comment (zyga) [inline debian/not-installed]: I didn't know there is a `debian/not-installed` file :-)

**Comment by zyga** [comment]
How do we generate a source tarball with vendored bits in place?

**Comment by vosst** [comment]
It seems like debian/not-installed is not used on trusty. The ppa builds are failing for the /usr/bin/uboot-go not being installed.

**Comment by mvo5** [comment]
Looking at the errors it appears that gccgo does not support vendoring. We need to figure out what is doing on here.

**Comment by mwhudson** [comment]
Why are you building with gccgo? Just for powerpc?

I can find you a terrible patch to make vendoring sort of work with gccgo...

**Comment by mwhudson** [comment]
https://github.com/tianon/debian-containerd/commit/560acae670d850e17876e79d6a437e77c6764733

**Comment by mvo5** [comment]
@mwhudson yeah, just for gccgo. Thanks a bunch for the patch!

**Comment by niemeyer** [comment]
Tests are broken.

**Comment by mvo5** [comment]
Test failures: `- Fetch and check assertions for snap "go-example-webserver" (16) (Get https://assertions.ubuntu.com/v1/assertions/snap-revision/SJyJUkh1WaZA4RzHwLtSt5WfWOqi9SWu5Zq5QIvMPgCoTJai2sydKX25o6YHyWIM: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers))` looks unrelated.

**Pull request merged**